swabbie-core/src/test/kotlin/com/netflix/spinnaker/swabbie/ResourceTypeHandlerTest.kt

```diff
@@ -659,6 +660,117 @@ object ResourceTypeHandlerTest {
     defaultHandler.deletionTimestamp(configuration) shouldMatch equalTo(plus2Days)
   }
 
+  @Test
+  fun `update delete timestamp should work`() {
+    val timestamp10days = clock.millis().plus(TimeUnit.DAYS.toMillis(10))
+    val timestamp1hour = clock.millis().plus(TimeUnit.HOURS.toMillis(1))
+    val resource1 = TestResource(
```

Comment:
test looks good functionally ... there might be some opportunity to reduce the verbosity of the setup.

```
val resource1 = appResource("1", "testResource-v001", app("testResource"), DAYS.toMillis(11))
val resource4 = imageResource("4", "my-package-0.0.2", package("my-package"), DAYS.toMillis(11))
```

motivation being to cut down on the amount of repeated boiler plate thus making it easier to grok the actual test intent.

Reply:
Going to refactor this whole test class in a separate PR. I agree it's hard to read, and there is way too much boilerplate.

Reply:
👍